x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_list/credentials_list.test.tsx

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import '../../../../__mocks__/kea.mock';
```

Comment:
v small nit: newline consistency

```suggestion
 */

import '../../../../__mocks__/kea.mock';
```

BTW, if you ever want me to just self-merge small nits like these so you don't have to deal with them, just let me know - I'm happy to!

Reply:
You could certainly do that if you'd like

d2e21d7407d6477e27304ee1903332e5179d534a